src/main/menus/ja/darwin.json

```diff
@@ -0,0 +1,80 @@
+{
+  "menu": [
```

Comment:
Sorry, forgot to update the documentation, the translation configuration of the menu has been moved to locale. I will update the translation guide right away.
Just translate all the files in the locale directory.
https://github.com/agalwood/Motrix/tree/master/src/shared/locales/en-US